src/flows/components/header/VersionHeader/CloneButton.tsx

```diff
@@ -0,0 +1,59 @@
+// Libraries
+import React, {FC, useContext} from 'react'
+import {useHistory} from 'react-router-dom'
+import {useSelector} from 'react-redux'
+
+// Contexts
+import {FlowContext} from 'src/flows/context/flow.current'
+
+// Components
+import {ComponentColor, SquareButton, IconFont} from '@influxdata/clockface'
+
+// Utility
+import {event} from 'src/cloud/utils/reporting'
+import {getOrg} from 'src/organizations/selectors'
+import {postNotebook} from 'src/client/notebooksRoutes'
+
+// Constants
+import {PROJECT_NAME_PLURAL} from 'src/flows'
+import {serialize} from 'src/flows/context/flow.list'
```

Comment:
not for this PR, but should we starting moving this out of the list context now that it's being used in places that have nothing to do with lists?